core/src/test/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreIT.java

```diff
@@ -2307,59 +2307,6 @@ private boolean waitForRelocationsToStart(final String index, TimeValue timeout)
         return awaitBusy(() -> client().admin().cluster().prepareHealth(index).execute().actionGet().getRelocatingShards() > 0, timeout.millis(), TimeUnit.MILLISECONDS);
     }
 
-    public void testBatchingShardUpdateTask() throws Exception {
```

Comment:
I removed this test as it uses the `BlockingClusterStateListener` which does not work with the new infrastructure. I will open a PR so that `SnapshotShardsService.innerUpdateSnapshotState` uses the default `ClusterService`-based batching mechanism, making this test obsolete.